app/src/main/res/values/strings.xml

```diff
@@ -304,6 +304,8 @@
     <string name="change_instance">Change instance</string>
     <string name="suggest_change_instance">Loading is taking more than usual. Consider changing instance</string>
     <string name="playback_pitch">Pitch</string>
+    <string name="playback_pitch_semitone">Semitone</string>
+    <string name="playback_pitch_semitone_error_input">Value must be between -12.0 (minimum) and 12.0 (maximum)</string>
```

Comment:
Please don't put the numbers inside the string, instead use `%1$.2f` and pass the argument with `getString(R.string.playback_pitch_semitone, 12f)`.

See https://www.w3schools.com/java/ref_string_format.asp.